portfolio/src/main/java/com/google/sps/servlets/JapanDataServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/japan-data")
+public class JapanDataServlet extends HttpServlet {
+
+  private LinkedHashMap<String, Integer> populationMap = new LinkedHashMap<>();
+
+  public void getData(Scanner scanner) {
```

Comment:
If getData is only being used in this class, then I would make this private instead of public. Err on the side of using private instead of public, unless the method will be referenced outside of its class